app.config.js

```diff
@@ -1,7 +1,12 @@
+const webpack = require('webpack')
 const IgnoreNotFoundExportPlugin = require('ignore-not-found-export-webpack-plugin')
 
 const appConfig = {
   plugins: [
+    new webpack.ProvidePlugin({
+      'cozy.client': 'cozy-client-js/dist/cozy-client.js',
+      'cozy.bar': 'cozy-bar/dist/cozy-bar.js'
```

Comment:
IIRC they are injected by the stack (see https://github.com/cozy/cozy-home/blob/master/src/targets/browser/index.ejs#L17), and they both expose themselves globally. I'm actually worried that this will expose them twice? And especially the bar, as of now, used the version provided by the stack and not the dependency of the app.

Reply:
I've reworked the feature here : https://github.com/cozy/cozy-home/pull/1490/commits/20b17eb38154df31865a78cb281008978649078d

In production, cozy-bar and cozy-client are from the stack assets, but in development, they are provided.